R/apa_print.R

```diff
@@ -32,4 +32,7 @@ apa_print <- function(x, ...) {
 # #' @importClassesFrom BayesFactor BFBayesFactor
 setGeneric("apa_print")
 
+#' @method apa_print default
```

Comment:
Hmm, I seem to remember reading in the `roxygen` documentation that the methods generally do not need to be specified manually. But I assume this is somehow necessary due to recent issues with method dispatch?

Reply:
Yes, this is what I also remembered. I searched for the source, it was Hadley Wickham's R packages book... however, end of last year he added a paragraph that disambiguation might be necessary and then `@method` should be used: http://r-pkgs.had.co.nz/man.html#man-classes
There also seems the something in the making with `@exportS3method`, but I could not find which version of `roxygen2` already supports it. I thought going with the stable version would be wise, especially because there seems to be changing so many thing at once, right now.

Reply:
Short answer: Yes, it's necessary. :-D

Reply:
If I understand correctly, `@exportS3method` was introduced in [`roxygen2` 7.0.0](https://github.com/r-lib/roxygen2/blob/4b501fe3e6ea8ca4e630f108d1ba8c84c55ac601/NEWS.md#roxygen2-700). So we should be able to use it, right?

Reply:
Didn't work on my machine, so I avoided it. :/

Reply:
I tried out `@exportS3Method` which now works (it was the capitalization that played a trick on me). Still, the effect is exactly the same as using `@method` and `@export` together. The only advantage of `@exportS3Method` seems to be that you are now able to write methods for generics from other packages more easily by using `@exportS3Method pkh::generic class`.

Reply:
Nice. So lets use `@exportS3Method` as it is the recommended approach and has a technical advantage.

Reply:
Would you mind merging this PR first and then I commit this change to the development branch in the main repository? My fork is already filled with many other things. :joy:

Reply:
Sure.